ibc/host/keys_ics05.go

```diff
@@ -0,0 +1,24 @@
+package host
```

Comment:
Thank you for splitting these into separate files!

Makes it so much easier to read and understand!